pkg/k8s/identitybackend/identity.go

```diff
@@ -39,6 +39,9 @@ const (
 
 	k8sPrefix               = labels.LabelSourceK8s + ":"
 	k8sNamespaceLabelPrefix = labels.LabelSourceK8s + ":" + k8sConst.PodNamespaceMetaLabels + labels.PathDelimiter
+
+	// byKeyIndex is the name of the by key index of the identities.
```

Comment:
nit: Reads a bit odd. 
```suggestion
	// byKeyIndex is the name of the index of the identities by key.
```

Reply:
Fixed